x-pack/elastic-agent/magefile.go

```diff
@@ -32,6 +32,8 @@ import (
 	// mage:import
 	_ "github.com/elastic/beats/v7/dev-tools/mage/target/integtest/notests"
 	// mage:import
+	_ "github.com/elastic/beats/v7/dev-tools/mage/target/systemtest"
```

Comment:
@jsoriano is it possible to include the command once, and apply it to all subprojects/beats?

Reply:
Not that I know, but maybe an idea is to add the target to `dev-tools/mage/target/integtest` (instead of to `dev-tools/mage/target/systemtest`), so it will be included in all projects that have go or python integration tests and already include these targets.

Reply:
I like the idea: will add a commit on top, thanks!